### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by MSancaktutar**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDM-8791
https://tools.hmcts.net/jira/browse/RDM-8796

### Change description ###
BEFTA integration
Removal of x-power-by headers
Bumped up of chart version
Introduced UV_THREADPOOL_SIZE: 64
fixes in smoke test setup

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[X] No
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # ---- Base Image ----
-ARG base=hmctspublic.azurecr.io/base/node:12-alpine
+
+ARG base=hmctspublic.azurecr.io/base/node:12-stretch-slim
```
Comment (timja) [inline Dockerfile]: ah no, revert this

File: .project [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (timja) [inline .project]: ?

File: .classpath [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (timja) [inline .classpath]: ?

File: aat/gradle/wrapper/gradle-wrapper.properties [hunk 4]
```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.3-all.zip
```
Comment (timja) [inline aat/gradle/wrapper/gradle-wrapper.properties]: this version is ancient, why would you use this on something new =/

File: web.config [hunk 5]
```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (timja) [inline web.config]: this is imminently moving to AKS so I recommend reverting this

File: Dockerfile (hunk 1 above)
Comment (MSancaktutar) [reply Dockerfile:3]: why so?

File: web.config (hunk 5 above)
Comment (MSancaktutar) [reply web.config:1]: is the above still valid? if so, why?

Comment (timja) [reply web.config:1]: yes still valid, this was moved to AKS months ago

File: .classpath (hunk 3 above)
Comment (MSancaktutar) [reply .classpath:1]: removed file

File: .project (hunk 2 above)
Comment (MSancaktutar) [reply .project]: removed

File: Dockerfile (hunk 1 above)
Comment (MSancaktutar) [reply Dockerfile]: sorry - merge mistake it turned out.

File: aat/gradle/wrapper/gradle-wrapper.properties (hunk 4 above)
Comment (MSancaktutar) [reply aat/gradle/wrapper/gradle-wrapper.properties:3]: You're right, but gradle upgrades will be taken care of as a separate activity. I just don't want to delay this pending such upgrade.

File: web.config (hunk 5 above)
Comment (MSancaktutar) [reply web.config]: file removed.

**Pull request merged**